Explorer/Assets/DCL/InWorldCamera/CameraReelGallery/CameraReelGalleryController.cs

```diff
@@ -141,16 +157,35 @@ async UniTaskVoid SetPublicFlagAsync(CancellationToken ct)
 
                 this.contextMenuController.ShareToXRequested += cameraReelResponse =>
```

Comment:
I personally would replace the inline lambdas with a named method to avoid allocations and allow the proper unsub in the disposal